private/numerics/functions/elliptic.rkt

```diff
@@ -0,0 +1,241 @@
+#lang racket/base
+
+;; Consult "Numerical Computation of Real or Complex Elliptic
+;; Integrals" Carlson (1994) http://arxiv.org/pdf/math/9409227v1.pdf
+;;
+;; The algorithm in scmutils seems similar to what is in that paper,
+;; but slightly different.
+;;
+;; Consider also exporting under the more familiar names Rf, Rj, Rc,
+;; Rd. Is there any input method that gets those names as subscripts.
+
+(require racket/math)
+(require racket/fixnum)
+
+(define machine-ε
+  (let loop ([ε 1.0])
+    (if (= 1.0 (+ ε 1.0))
+        (* 2 ε)
+        (loop (/ ε 2)))))
+
```

Comment:
Also, does Racket have a built-in to compute the machine ε? I searched for a while but could find it.